spikingjelly/activation_based/triton_kernel/flex_sn_inductor/kernel.py

```
@@ -0,0 +1,174 @@
+
+    The template calls bwd_fn(saved..., grad_out0..., grad_state0...) for ALL
+    outputs and states.  AOT autograd drops gradient arguments for non-
+    differentiable outputs (e.g. spike signals from hard threshold), so the
+    actual backward function may accept fewer arguments.  This shim accepts
+    the full template signature and forwards only the used arguments.
+    """
+    all_grads = (["gs_" + str(i) for i in range(num_outputs)] +
+                 ["gv_" + str(i) for i in range(num_states)])
+    shim_args = ["sv_" + str(i) for i in range(n_saved)] + all_grads
+    fwd_call  = (["sv_" + str(i) for i in range(n_saved)] +
+                 [name for name, d in zip(all_grads, diff_mask) if d])
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)